### ISSUE TITLE
Request for original training data (questions + images + options + labels)

### ISSUE THREAD

**Issue by yandxdepp** [SII-zyj/Ophiuchus#11]
Hello! I have read your paper with great interest, and I am very impressed by your work. For a research project I am working on, I would like to request access to the original training dataset used in your paper.  I only need the raw data – questions, images, options, and ground-truth labels. Thanks!
